js/main.js

```diff
@@ -0,0 +1,24 @@
+const menuBars = document.querySelector('.menu-bars');
+const closeMenu = document.querySelector('.menu-close');
+const nav = document.querySelector('.desktop-nav');
+const navLinks = document.querySelectorAll('.nav-link');
+
+const closeMobileNav = () => {
+  nav.classList.remove('open');
+};
+
+console.log('navLink');
```

Comment:
It is good to use console.log for debugging, but it has to be removed. Thus your code will comply with the best industry practices. Please remove all consoles in your code.